src/components/material/digital-modal/helper.ts

```diff
@@ -0,0 +1,15 @@
+import { DigitalArticleService } from "../../../core/dbc-gateway/generated/graphql";
+import { Manifestation } from "../../../core/utils/types/entities";
+
+export const createDigitalModalId = (digitalArticleIssn: string) =>
+  `digital-modal-${digitalArticleIssn}`;
```

Comment:
I think we should make a string literal type for the issn id. Like we for instance have for FaustId.